test/test.js

```diff
@@ -16,6 +16,7 @@ const http = require('http');
 const fs = require('fs');
 const path = require('path');
 const stream = require('stream');
+const buffer = require('buffer');
```

Comment:
Is this used anywhere?

Reply:
Nope. Accidentally left over from an earlier iteration. Removed!